model/project_ref.go

```diff
@@ -1787,10 +1787,15 @@ func SaveProjectPageForSection(projectId string, p *ProjectRef, section ProjectP
 			ProjectRefDisabledStatsCacheKey:      p.DisabledStatsCache,
 			ProjectRefFilesIgnoredFromCacheKey:   p.FilesIgnoredFromCache,
 		}
-		if !isRepo && !p.UseRepoSettings() {
+		if !isRepo && !p.UseRepoSettings() && !defaultToRepo {
 			setUpdate[ProjectRefOwnerKey] = p.Owner
 			setUpdate[ProjectRefRepoKey] = p.Repo
 		}
+		if !defaultToRepo {
+			setUpdate[ProjectRefEnabledKey] = p.Enabled
```

Comment:
Enabled actually is something you should be able to default to repo so I don't think this should be included here.

Reply:
So if a repo is disabled, and a project is defaulted to the repo, then the project will become disabled. Is that what we want?

Reply:
(done)